### DISCUSSION TITLE
DSPy tracing

### DISCUSSION THREAD

**Discussion by paolotamag** [langfuse/langfuse#2605] [Ideas]
### Describe the feature or potential improvement

can we have something like this for Langfuse?

https://colab.research.google.com/github/Arize-ai/phoenix/blob/main/tutorials/tracing/dspy_tracing_tutorial.ipynb

**Comment by marcklingen** [comment]
I’ve just added a notebook featuring the Langfuse integration that was added to dspy: https://langfuse.com/docs/integrations/dspy
